fiatconnect-api.md

```diff
@@ -383,105 +399,201 @@ The FiatConnect API specifies *two* types of authentication; the first authentic
+
+The first group of endpoints are those that *do not* require the user to have a logged-in session. These are:
 
 * `GET /quote/in`
 * `GET /quote/out`
+* `POST /auth/login`
+
+Since it may be a painful user experience to have a user log in to a provider just to get a quote, authentication is not required for these endpoints. Of course,
+clients may still access these endpoints when logged in, but a logged-in session (or any session at all) is not required to access these endpoints. `POST /auth/login`
+does not require a user to already be logged in, as described above.
+
+##### 3.2.1.6.2. Privileged Endpoints
+
+The second, and largest set of endpoints, are the ones that require a user to be logged in. These endpoints are the ones that deal with reading and writing user-specific data.
+
+* `POST /kyc/:kycSchema`
 * `GET /kyc/:kycSchema/status`
+* `DELETE /kyc/:kycSchema`
+* `POST /accounts/:fiatAccountSchema`
+* `GET /accounts`
+* `DELETE /account/:fiatAccountId`
 * `GET /transfer/:transferId/status`
+* `POST /transfer/in`
+* `POST /transfer/out`
+
+For these endpoints, a user needs to be logged in with an authenticated session. This session may have the maximum expiration time allowed by the FiatConnect standard (four hours).
+
+If a client attempts to access any of these endpoints without a session, the server MUST return a `401` status code with an `Unauthorized` error.
+If a client tries to acces any of these endpoints with an expired session, the server MUST respond with an HTTP `401` status code with a `SessionExpired` error.
 
-The rationale behind not requiring an `exp` claim for these endpoints is that these endpoints are *non-mutating*; calling them does not affect server-side
-state in any way. Since these endpoints are less security-critical, an `exp` claim is not required as a convenience to clients.
-While these endpoints do not require the `exp` claim to be present, *all endpoints* MUST honor the `exp` claim *if it is* present in the
-JWT payload. For endpoints where the `exp` claim is required, the server MUST return an HTTP `400` if it is not present.
-A server MUST return an HTTP `401` error if the provided `exp` field is later than the current time with respect to Epoch. If an `exp` claim is
-included in a request, it MUST NOT exceed 24 hours from the current time with respect to Epoch. If the `exp` claim is more than 24 hours into the future, the
-server MUST return an HTTP `400` error. This is required in order to prevent clients from generating excessivley long-lived JWTs that would pose a security
-risk if leaked.
+### 3.2.2. Client API Key
 
-##### 3.2.1.2.4. `"client"` Claim
+In addition to the SIWE-based auth used to authenticate users and create sessions, the FiatConnect specification also supports a more traditional authentication standard, used
+*only* to identify specific clients with requests. Recall the webhook-based status monitoring mentioned earlier in this document. In order to support status
```

Comment:
```suggestion
 to identify specific clients with requests. Recall the webhook-based status monitoring mentioned earlier in this document. In order to support status
```
to say here sounds like it may be too prescriptive. If another use for it was found that didn't break the spec in some other way, I don't think we should box people in

Reply:
Sure, I think that makes sense!